click_odoo_contrib/update.py

```diff
@@ -176,8 +176,8 @@ def _get_checksum_dir(cr, module_name):
 
 def _is_installable(module_name):
     try:
-        if odoo.tools.parse_version(odoo.release.version) <= odoo.tools.parse_version(
-            "15"
+        if odoo.tools.parse_version(odoo.release.version) < odoo.tools.parse_version(
+            "16dev"
         ):
```

Comment:
```suggestion
        if odoo.release.version_info < (16, 0):
```

@kafai-lam I think this is the best way. Can you try?